instance_test.go

```diff
@@ -235,8 +248,8 @@ func TestDeleteInstance(t *testing.T) {
 
 func TestRebootInstance(t *testing.T) {
 	client, server, _ := NewAdvancedClientForTesting(map[string]map[string]string{
-		"/v2/instances/12345/hard_reboots": map[string]string{
-			"requestBody":  ``,
+		"/v2/instances/12345/hard_reboots": {
+			"requestBody":  `""`,
```

Comment:
Still think we should replace the body if it's JSON with `{}` if it's `""`. Not sure Rails will handle an empty string as a JSON body.

Reply:
mm, i can try, let me check this

Reply:
@andyjeffries All good i try this:

`resp, err := client.RebootInstance("id_instances")`

and the return was:

`&{success   }`

Reply:
OK then :)